types/express-paginate/express-paginate-tests.ts

```diff
@@ -0,0 +1,26 @@
+import * as express from 'express';
```

Comment:
Since this is a callable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
Can you please give me a resource to read about this illegality?
I somehow understand, but would like a resource to better understand this.

Reply:
Sure, check this out: https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this